pkg/proc/variables.go

```diff
@@ -1264,6 +1263,15 @@ func (v *Variable) loadValueInternal(recurseLevel int, cfg LoadConfig) {
 	switch v.Kind {
 	case reflect.Ptr, reflect.UnsafePointer:
 		v.Len = 1
+
```

Comment:
Rather than reading the value twice use the value of v.Children[0].Addr.

Reply:
> Done. But this is incorrect if the maybeDereference() function took the early return, returning itself

the early return that says `if v.Unreadable != nil`? We return early from loadValueInternal anyway if unreadable is set.